File: fix_indent.py
import re

with open('src\\AppStyles.tsx', 'r', encoding='utf-8') as f:
    content = f.read()

# Fix the broken indentation on .hero-section::before lines in mobile media queries
content = re.sub(
    r'^(\s*\.hero-section::before \{)\s*$',
    r'      .hero-section::before {',
    content,
    flags=re.MULTILINE
)

with open('src\\AppStyles.tsx', 'w', encoding='utf-8') as f:
    f.write(content)

print('Indentation fixed!')

File: fix_indent.js
const fs = require('fs');
const filePath = 'src/AppStyles.tsx';
let content = fs.readFileSync(filePath, 'utf8');

// Fix broken indentation on .hero-section::before lines
content = content.replace(
  /^\s+\.hero-section::before \{/gm,
  '      .hero-section::before {'
);

fs.writeFileSync(filePath, content, 'utf8');
console.log('Indentation fixed!');
